app/tests/integration/e2e_test_variables.json

```diff
@@ -24,5 +24,53 @@
                 "firefox_pref_branch": "default"
             }
         ]
+    },
+    "multi-pref-experiment": {
+        "action_name": "multi-preference-experiment",
+        "name": "E2E experimenter multipref-test",
+        "type": "channel",
+        "channels": "Nightly",
+        "min_version": "80.0",
+        "max_version": "100.0",
+        "userFacingName": "e2e testing name",
+        "userFacingDescription": "e2e testing description",
+        "branches": [
+            {
+                "branch_description": "multipref-branch-description",
+                "branch_name": "e2e-multipref-branch",
+                "preferences": [
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-1",
+                        "firefox_pref_type": "boolean",
+                        "firefox_pref_value": "true"
+                    },
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-1-1",
+                        "firefox_pref_type": "boolean",
+                        "firefox_pref_value": "false"
+                    }
+                ]
+            },
+            {
+                "branch_description": "multipref-branch-description-2",
+                "branch_name": "e2e-multipref-branch-2",
+                "preferences": [
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-2",
+                        "firefox_pref_type": "boolean",
+                        "firefox_pref_value": "false"
+                    },
+                    {
+                        "firefox_pref_branch": "default",
+                        "firefox_pref_name": "e2e-testing-branch-2-1",
+                        "firefox_pref_type": "boolean",
+                        "firefox_pref_value": "true"
+                    }
```

Comment:
Thank you for adding multiple preferences!  One last thing, can you make them each one of each of the four types we support:

1) boolean
2) int
3) str
4) json str

Just so we can hit all four cases in one test?